es.hjson

```diff
@@ -14,111 +14,111 @@
     "joinLockExceptionAddedAnswerText": "Excepción añadida"
 
-    "chatNameRemoved": "<b>El Nombre de chat eliminado por </b>%s"
-    "chatNameRemovedAnswerText": "Nombre de chat eliminado"
-    "chatNameNotSet": "<b>El nombre del chat no se configuró, por lo que no hay nada que eliminar.</b>"
-    "chatNameNotSetAnswerText": "Nombre de chat no establecido"
+    "chatNameRemoved": "<b>Nombre del chat eliminado por </b>%s"
+    "chatNameRemovedAnswerText": "Nombre del chat eliminado"
+    "chatNameNotSet": "<b>El nombre del chat no estaba establecido, así que no hay nada que eliminar.</b>"
+    "chatNameNotSetAnswerText": "Nombre del chat no establecido"
 
     "cantUseFeatureOnAdminsAnswerText": "No puede %s administradores"
 
-    "CalsiBotNoLongerAdmin": "<b>El CalsiBot ya no es administrador o no tiene suficientes permisos, por lo que este teclado ha caducado.</b>"
-    "CalsiBotNoLongerAdminAnswerText": "CalsiBot ya no es administrador o no tiene suficientes permisos, por lo que este teclado ha caducado."
+    "CalsiBotNoLongerAdmin": "<b>CalsiBot ya no es un administrador o no tiene suficientes permisos, por lo que esta botonera ha expirado.</b>"
+    "CalsiBotNoLongerAdminAnswerText": "CalsiBot ya no es un administrador o no tiene suficientes permisos, por lo que esta botonera ha expirado."
 
-    "keyboardHasExpired": "<b>El Este teclado ha caducado.</b>"
-    "keyboardExpiredAnswerText": "Teclado caducado"
-    "keyboardUpdatedPressConfirmAnswerText": "Teclado actualizado, presione 'Confirmar' para aplicar"
+    "keyboardHasExpired": "<b>Esta botonera ha caducado.</b>"
+    "keyboardExpiredAnswerText": "Botonera caducada"
+    "keyboardUpdatedPressConfirmAnswerText": "Botonera actualizada, presione 'Confirmar' para aplicar"
   }
 
   "generic.buttons": {
     "whatIsThis": "❓ ¿Qué es esto?"
     "whatIsThisShort": "❓ ¿Qué?"
-    "imSorry": "😭 Lo siento!"
+    "imSorry": "😭 ¡Lo siento!"
 
     "joinChannel": "👋 Unirse al Canal"
     "addToGroup": "🔱 Añadir al Grupo"
     "tapToVerify": "🤖 Toca para verificar 🤖"
 
-    "banUser": "Prohibir usuario"
-    "banBot": "Ban Bot"
-    "allowUser": "Permitir usuario"
+    "banUser": "Banear usuario"
+    "banBot": "Banear Bot"
+    "allowUser": "Autorizar al usuario"
     "trustUser": "Usuario de confianza"
-    "trustBot": "Trust Bot"
-    "unbanUser": "Usuario de Unban"
+    "trustBot": "Bot de confianza"
+    "unbanUser": "Desbanear usuario"
 
-    "settings": "⚙️ Configuración"
+    "settings": "⚙️ Ajustes"
 
     "resetRestrictions": "🤝 Restablecer restricciones"
-    "yes": "✅ sí"
+    "yes": "✅ Sí"
     "no": "❌ No"
-    "enable": "✅ Habilitar"
+    "enable": "✅ Activar"
     "disable": "❌ Desactivar"
     "disableLimit": "❌ Desactivar límite"
     "disableTimeLimit": "❌ Desactivar límite de tiempo"
 
-    "permabanUser": "❌ Usuario de Permaban"
-    "unPermabanUser": "🤝 Usuario no permanente"
+    "permabanUser": "❌ Banear usuario permanentemente"

```

Comment:
This text will be shown on an inline button, so there is very limited space.

Is there a way we can shorten this, even if it involves some informal language?